feeds/wlan-ap/opensync/src/platform/openwrt/src/command/src/task.c

```diff
@@ -165,8 +170,35 @@ static int command_conf_add(struct schema_Command_Config *conf)
 	return -1;
 }
 
-static void command_conf_del(struct schema_Command_Config *conf)
+static int command_conf_del(struct schema_Command_Config *conf)
 {
+	int ret=0;
+	struct task *task=NULL, *next=NULL;
+	int err;
+	json_t* where;
+
+	list_for_each_entry_safe(task, next, &tasks,list) {
+		if (strcmp(task->conf._uuid.uuid, conf->_uuid.uuid)) {
+			continue;
+		}
+		if(!strcmp(task->state,"running")) {
+			LOGI("Deletes running Command not allowed:%s %s %s",task->conf.command,task->conf._uuid.uuid,task->state);
+			ret = -1;
+			break;
+		}
+		LOGI("Delete scheduled Command:%s ,UUID:%s %s",task->conf.command,task->conf._uuid.uuid,task->state);
```

Comment:
This log should be LOGD.  Mostly we don't care.

Reply:
yes, I'll do that